.github/workflows/build-dockerfile.yml

```diff
@@ -0,0 +1,17 @@
+name: Build-Dockerfile
+
+on: pull_request
+
+jobs:
+  build-dockerfile:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+
+      - name: Build the Dockerfile
+        run: docker build -f examples/Dockerfile/Dockerfile -t tiledb:dev .
+        
+      - name: Run the built docker image
+        run: docker run tiledb:dev
```

Comment:
I don't think this does anything does it? The entrypoint is just bash?

Reply:
Right, but it exists in the existing travis-CI: https://github.com/TileDB-Inc/TileDB-Docker/blob/master/.travis.yml

We can remove it, but I wasn't sure if there might be a scenario where running the image might reveal an issue in the build. Probably not.

Reply:
Removed